report.qmd

```diff
@@ -1,60 +1,61 @@
+  - name: "Teun van Dijk"
+    affiliation: "MSc Urbanism"
+    role: "Presentation & design lead"
+
 format: html
 ---
 
-::: callout
-## This is a computational notebook
-
-Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.
-
-## Running Code
+| Name                   | Affiliation     | Role                               |
+|------------------------|-----------------|------------------------------------|
+| Daan Schlosser         | MSc Geomatics   | Project coordinator & data-analyst |
+| Joost Bastiaansen      | MSc Urbanism    | Mapping specialist                 |
+| Aparnaa Chandrasekaran | MSc Urbanism    | Research lead                      |
+| Teun van Dijk          | MSc Urbanism    | Presentation & design lead         |
 
-When you click the **Render** button a document will be generated that includes both content and the output of embedded code. You can embed code like this:
+<br>
 
-```{r}
-1 + 1
-```
+## Introduction
 
-You can add options to executable code like this 
+### Site Selection
+This project will focus on the city of Dresden. In order to provide an answer to the main research question “what areas of Dresden would stand to gain the most from the urban stream restoration project?” a division of areas has to be provided. To start, a grid with tiles of 500 by 500 metres was defined for the city of Dresden. This is filtered to only show the tiles which contain a stream to remove any unnecessary information.
 
-```{r}
-#| echo: false
-2 * 2
-```
+![Step 1: A 500 by 500 metres grid covering Dresden](images\introduction\STEP1.png)
 
-The `echo: false` option disables the printing of code (only output is displayed).
 
-You can delete this entire callout between and including the lines
-```
-::: callout
-```
-and
+![Step 2: Overlay with streams](images\introduction\STEP2.png)
 
-```
-:::
-```
-and start the report from the Introduction section below.
-:::
 
-## Introduction
+![Step 3: Remove any unecessary information](images\introduction\STEP3.png)
 
 ## Methods
 
 ### MCDA
 
+#### Quality of Life
+![Physical Health: Vicinity to Urban Green Space](images\quality_of_life\VicinityToUrbanGreenSpace.jpeg)
+![Physical Health: Vicinity to Urban Green Space](images\quality_of_life\VicinityToUrbanGreenSpace_Grid.jpeg)
+![Mental Health: Vicinity to Stream](images\quality_of_life\VicinityToStream.jpeg)
+![Mental Health: Vicinity to Stream](images\quality_of_life\VicinityToStream_Grid.jpeg)
+![General Health: Facilities](images\quality_of_life\Facilities.jpeg)
+![General Health: Facilities](images\quality_of_life\Facilities_Grid.jpeg)
+![Accessibility: Local Integration](images\quality_of_life\LocalIntegration.jpeg)
+![Accessibility: Local Integration](images\quality_of_life\LocalIntegration_Grid.jpeg)
+![Accessibility: Appearance](images\quality_of_life\Covered.jpeg)
+![Accessibility: Appearance](images\quality_of_life\Covered_Grid.jpeg)
```

Comment:
All these should be in the results section. The methods section is meant to show _how_ the analysis was carried out.